config.example.yml

```diff
@@ -23,6 +23,28 @@ database:
   # Database password.
   password: CHANGEME
 
+  # List of low-level database options that can be set to influence some Icinga DB internal default behaviours.
+  # Do not change the defaults if you don't have to!
+  options:
+    # Maximum number of connections Icinga DB is allowed to open in parallel to the database.
+    # By default, Icinga DB is allowed to open up to "16" connections whenever necessary.
+#    max_connections: 16
+
+    # Maximum number of queries allowed to connect to a single database table simultaneously.
+    # By default, Icinga DB is allowed to execute up to "8" queries of any kind, e.g. INSERT,UPDATE,DELETE
+    # concurrently on a given table.
```

Comment:
Please note for this and the following options that a positive integer is expected.